rundeckapp/grails-app/domain/rundeck/Option.groovy

```diff
@@ -311,10 +329,12 @@ public class Option implements Comparable{
      */
     void convertValuesList(){
         if(valuesList!=null){
-            def x=new TreeSet()
-            x.addAll(valuesList.split(",").collect{it.trim()}.grep{it} as List)
+            if(valuesListDelimiter == null){
+                valuesListDelimiter=DEFAULT_DELIMITER
+            }
+            def x=new ArrayList()
+            x.addAll(valuesList.split(valuesListDelimiter).collect{it.trim()}.grep{it} as List)
```

Comment:
`valuesList.split(valuesListDelimiter)` if the delimiter is a regex-special char this would not work.  Maybe `valuesList.split(Pattern.quote(valuesListDelimiter))`